lib/charms/layer/nodejs.py

```diff
@@ -12,11 +12,10 @@ def node_dist_dir():
     Returns:
     Absolute string of node application directory
     """
-    storage_id = hookenv.storage_list('app')[0]
-    return hookenv.storage_get('location', storage_id)
```

Comment:
I'm not sure. Why don't I run a few tests and see what happens ... possibly an upgrade test of deploying ghost with the current master, then building ghost with my changes and upgrading.. things in this area.